### ISSUE TITLE
Attributes template isn't rendering properly

### ISSUE THREAD

**Issue by lornemcintosh** [balderdashy/sails-generate-model#5]
I'm just trying Sails for the first time, and I'm getting some strange model output from sails-generate-model. For example, this command:
```
sails generate model User name:string email:string
info: Created a new model ("User") at api/models/User.js!
```
creates this output in api/models/User.js:
```javascript
/**
* User.js
*
* @description :: TODO: You might write a short summary of how this model works and what it represents here.
* @docs        :: http://sailsjs.org/#!documentation/models
*/

module.exports = {

  attributes: {
function (obj) {
obj || (obj = {});
var __t, __p = '', __j = Array.prototype.join;
function print() { __p += __j.call(arguments, '') }
with (obj) {


/**
 * Boilerplate attribute
 *
 * This is the template used when creating attributes
 * for models generated through the CLI
 * e.g. `sails generate model foo attrName`
 *
 */
;
__p += '\n';
if (lang === 'js') {;
__p += '    ' +
((__t = ( name )) == null ? '' : __t) +
' : { type: \'' +
((__t = ( type )) == null ? '' : __t) +
'\' }\n';
} else if (lang === 'coffee'){ ;
__p += '    ' +
((__t = (name)) == null ? '' : __t) +
':\n      type: \'' +
((__t = ( type )) == null ? '' : __t) +
'\'\n';
};
__p += '\n';

}
return __p
},
function (obj) {
obj || (obj = {});
var __t, __p = '', __j = Array.prototype.join;
function print() { __p += __j.call(arguments, '') }
with (obj) {


/**
 * Boilerplate attribute
 *
 * This is the template used when creating attributes
 * for models generated through the CLI
 * e.g. `sails generate model foo attrName`
 *
 */
;
__p += '\n';
if (lang === 'js') {;
__p += '    ' +
((__t = ( name )) == null ? '' : __t) +
' : { type: \'' +
((__t = ( type )) == null ? '' : __t) +
'\' }\n';
} else if (lang === 'coffee'){ ;
__p += '    ' +
((__t = (name)) == null ? '' : __t) +
':\n      type: \'' +
((__t = ( type )) == null ? '' : __t) +
'\'\n';
};
__p += '\n';

}
return __p
}
  }
};
```

Obviously the attributes template isn't rendering properly, but I have no idea why. I'm using Node 0.12.0, Sails 0.11.0, sails-generate 0.12.2, and sails-generate-model 0.10.9 on Windows 7 64-bit. Let me know if I can provide any more relevant info. Thanks!

**Comment by sgress454** [comment]
Thanks for the report @lornemcintosh!  We'll check it out.  In the meantime, try deleting that file and running `sails generate model User` without setting attributes on the command line?

**Comment by lornemcintosh** [comment]
Yes, it works as expected without the attributes.

**Comment by sailsbot** [comment]
Thanks for posting, @lornemcintosh.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by chadyred** [comment]
Close but always the same problem !

```
module.exports = {

  attributes: {
function (obj) {
obj || (obj = {});
var __t, __p = '', __j = Array.prototype.join;
function print() { __p += __j.call(arguments, '') }
with (obj) {


/**
 * Boilerplate attribute
 *
 * This is the template used when creating attributes
 * for models generated through the CLI
 * e.g. `sails generate model foo attrName`
 *
 */
```

**Comment by konstantinzolotarev** [comment]
What version of sails do you use ?

**Comment by konstantinzolotarev** [comment]
@chadyred What version of Sails do you use ?
